ntbs-service/wwwroot/css/site.scss

```diff
@@ -147,3 +147,18 @@ body {
 .flex-container {
     display: flex;
 }
+
+.alerts-heading {
+    height: 100%;
+    margin: auto 0;
+}
+
+.tb-service-alerts-filter-label {
+    height: 100%;
+    margin: auto 0 auto auto;
+}
+
+.tb-service-alerts-filter-dropdown {
+    max-width: 200px;
+    margin: 5px 0 auto 15px
+}
```

Comment:
I think this ought to be in a seperate file, either index page specific, or table display specific